public/static/js/spawned_hosts.js

```diff
@@ -16,13 +16,19 @@ mciModule.controller('SpawnedHostsCtrl', ['$scope', '$window', '$timeout', '$q',
     $scope.curVolumeData;
     $scope.maxHostsPerUser = $window.maxHostsPerUser;
     $scope.maxUnexpirableHostsPerUser = $window.maxUnexpirableHostsPerUser;
+    $scope.maxVolumeSizePerUser = $window.maxVolumeSizePerUser;
     $scope.spawnReqSent = false;
     $scope.useTaskConfig = false;
     $scope.isVirtualWorkstation = false;
     $scope.noExpiration = false;
-    $scope.homeVolumeSize = 500;
+    $scope.volumeSizeDefault = 100;
```

Comment:
500 was Sam's estimate of a typical server developer's space requirements
(https://github.com/evergreen-ci/evergreen/pull/3021#issuecomment-572781936)

Reply:
Okay, I picked 100 because our current max volume size for users is 200 in prod right now. I'm going to bump it up then.

Reply:
should we also change the [default](https://github.com/evergreen-ci/evergreen/blob/b9fe49b09c8a11eef44f517b21de4e28f4cd98c5/model/host/host.go#L328)?